pilot/pkg/model/config_test.go

```diff
@@ -582,6 +591,46 @@ func TestAuthenticationPolicyConfig(t *testing.T) {
 			},
 			PrincipalBinding: authn.PrincipalBinding_USE_ORIGIN,
 		},
+		"mockonce": {
```

Comment:
as we have more test cases, let's change the name to be a bit more descriptive

Reply:
updated